internal/compiler/object_tree.rs

```diff
@@ -319,6 +319,7 @@ pub struct Timer {
     pub interval: NamedReference,
     pub triggered: NamedReference,
     pub running: NamedReference,
+    pub element: ElementRc,
```

Comment:
My guts feeling is that this should perhaps be an `ElementWeak` instead of an `ElementRc`.

Reply:
indeed, to be sure, it should be an ElementWeak.